datanode/entities/fund_period.go

```diff
@@ -0,0 +1,202 @@
+package entities
```

Comment:
Could you rename the file to `funding_period.go`?

We might have fund accounts eventually and they might have some periods, so this might end up being confusing.

Reply:
Yes, sorry, that was a typo, I will fix.